### PR TITLE
Update the contrib grafana dashboard

### PR DESCRIPTION
**Pull request by erikjohnston**
This is just a dumb import from our internal grafana.

### CODE REVIEW

File: contrib/grafana/synapse.json [hunk 1]
```diff
@@ -7,14 +7,22 @@
...
       "pluginId": "prometheus",
       "pluginName": "Prometheus"
+    },
+    {
+      "name": "DS_JKI.RE",
+      "label": "jki.re",
+      "description": "",
+      "type": "datasource",
+      "pluginId": "prometheus",
+      "pluginName": "Prometheus"
```
Comment (anoadragon453) [inline contrib/grafana/synapse.json:17]: You probably don't want to add this bit in :slightly_smiling_face:

File: contrib/grafana/synapse.json [hunk 2]
```diff
@@ -6635,107 +9892,29 @@
...
-          "datasource": "$datasource",
-          "description": "Number of rooms with the given number of forward extremities or fewer.\n\nThis is only updated once an hour.",
-          "fill": 0,
+          "datasource": "${DS_JKI.RE}",
+          "fieldConfig": {
+            "defaults": {
+              "custom": {}
+            },
+            "overrides": []
+          },
```
Comment (anoadragon453) [inline contrib/grafana/synapse.json:9907]: And here.

**Review by anoadragon453 — COMMENTED** [review]
A little bit of extra stuff crept in there, but lgtm otherwise. Thanks for doing this!

File: contrib/grafana/synapse.json (hunk 1 above)
Comment (erikjohnston) [reply contrib/grafana/synapse.json:17]: Woops, thanks!

Comment (erikjohnston) [reply contrib/grafana/synapse.json]: (this was because one of the graphs had accidentally been set to always be for `jki.re`)

**Comment by erikjohnston** [comment]
I fixed it in grafana and have reexported it again

**Review by anoadragon453 — APPROVED** [review]
lgtm then!

**Pull request merged**